src/PillSelect.tsx

```diff
@@ -1,6 +1,6 @@
 import { Menu } from '@headlessui/react';
-import clsx from 'clsx';
-import { useState } from 'react';
```

Comment:
I think React should stay here in each file in the component library (only next.js does not need it)
I think this should also trigger a lint warning